source/modulo_components/package.xml

```diff
@@ -4,22 +4,17 @@
   <name>modulo_components</name>
   <version>0.0.1</version>
   <description>TODO</description>
-  <maintainer email="baptiste@aica.tech">TODO</maintainer>
```

Comment:
Haha these are the laziest TODOs I've ever seen, thanks for spotting and resolving them!